resources/docker/workspace/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM php:8.0-cli-alpine
```

Comment:
interesting stuff! Thanks for adding this @JeroenVanOort 

Do you use it with your editor?